ci/aks/aks-cluster.lokocfg.envsubst

```diff
@@ -38,7 +38,7 @@ variable "workers_type" {
 }
 
 variable "location" {
-  default = "Germany West Central"
+  default = "West US 2"
```

Comment:
different PR ?

Reply:
Since they are in different commit, they should be ok. I think creating a different PR will only increase time to fix this issue. If AWS passes we can merge this PR immediately, no?

Reply:
I don't mind. Its not a big change.

Reply:
Thanks Imran.